core/types.go

```diff
@@ -4,6 +4,7 @@ package core
 
 import (
 	"context"
+	"database/sql"
```

Comment:
It seems unnatural to take a sql dependency here. 
I think instead you could move `ExpirationToTime` to the database package, since that is the only place it is called.

Reply:
It went into types to mirror `UnixTimeMsToTime`, but having it in database makes sense. I'll move it over.